src/mainwindow.cpp

```diff
@@ -231,8 +231,10 @@ void MainWindow::saveAs()
 
 void MainWindow::displayLeagues()
 {
-    if (apDbConnection->isOpen())
-        apDbConnection->displayLeagues();
+    for (unsigned int i = 0; i < aLeaguesId.size(); ++i)
```

Comment:
const unsigned int leagueSize = aLeaguesId.size();
for (unsigned int i = 0; i < leagueSize; ++i)
Just to optimize ;)

Reply:
Changed